access_control.py

```diff
@@ -0,0 +1,225 @@
+"""
+This script will process the `grant_access/` and `revoke_access/` requests.
+
+Main logic lives in conda-smithy. This is just a wrapper for admin-requests infra.
+"""
+import os
+import subprocess
+import sys
+import tempfile
+from pathlib import Path
+
+import requests
+from ruamel.yaml import YAML
+
+GH_ORG = os.environ.get("GH_ORG", "conda-forge")
+
+CIRUN_FILENAME_RESOURCE_MAPPING = {
```

Comment:
This is the configuration dictionary that maps `{grant,revoke}_access/<name>.txt` to an actual resource in ... `.access_control.yml`? Or where? Can you document the expected schema with a `#: comment` on top?